app/ios/STA/Info.plist

```diff
@@ -21,7 +21,7 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>3</string>
```

Comment:
Should this be `4` now if it should match the `versionCode` [here](https://github.com/Scottish-Tech-Army/Volunteer-app/blob/main/app/android/app/build.gradle#L132)?

Reply:
We can try to keep the Android and iOS build numbers in sync but in my experience, they usually tend to diverge a bit as we'll likely need to make separate builds for each platform. The semantic version we should definitely keep in sync though.

Reply:
Ah right, cool -- that's handy to know! 👍

Reply:
In the draft [Android deployment PR](https://github.com/Scottish-Tech-Army/Volunteer-app/pull/84) that I'm working on, I'm currently taking the app/package.json to be the single source of truth as to which semantic version we're up to.

Is the semantic version also needed in the iOS deployment process, and if so, where does it get it from? Does pulling it from package.json work for keeping it in sync across the two platforms?